packages/matter.js/package.json

```diff
@@ -25,7 +25,10 @@
   },
   "scripts": {
     "clean": "rm -rf dist && rm -rf out && rm *.tsbuildinfo || true",
-    "build": "tsc --project tsconfig.dist-es.json && tsc --project tsconfig.dist-cjs.json && tsc --project tsconfig.dist-dts.json",
+    "build-cjs": "tsc --project tsconfig.dist-cjs.json && cp package-cjs.json dist/cjs/package.json",
```

Comment:
Maybe the copy can be done by adding this file in the tsconfig.dist-cjs.json as part of the include ?

Reply:
Thoisis with JSON "can" work but only if you also add "resolveJsonModule" to true which I'm not sure we want (I would propose not). So the cp is the best solution for now